mlrun/config.py

```diff
@@ -50,6 +50,10 @@
     "version": "",  # will be set to current version
     "images_tag": "",  # tag to use with mlrun images e.g. mlrun/mlrun (defaults to version)
     "images_registry": "",  # registry to use with mlrun images e.g. quay.io/ (defaults to empty, for dockerhub)
+    # comma separated list of images that are in the specified images_registry, and therefore will be enriched with this
+    # registry when used. default to empty which means any image which is of the mlrun repository (mlrun/mlrun,
+    # mlrun/ml-base, etc...)
```

Comment:
empty == enrich mlrun/*
means there's no way to configure it to never enrich
use something like `mlrun/*` as default to have flexibility to have it never enrich as well when it's needed (open source?)

Reply:
There is a way to never enrich - not configuring the `images_registry`
This config value is relevant only when enriched is wanted - i.e. `images_registry` has a value

Reply:
That's not desirable - can't disable an enrichment by configuration, but just use the value to make it ineffectual